packages/sitecore-jss/package.json

```diff
@@ -47,6 +47,7 @@
   },
   "dependencies": {
     "axios": "^0.21.1",
+    "chalk": "^4.1.0",
```

Comment:
This dependency was missed in this PR #631 . Angular sample was throwing build error